### PR TITLE
Added Readme.md

### LINKED ISSUE #8: [📑 Docs]: Add proper README and CONTRIBUTING.md files
Improve the README of this repository by adding proper tags and `Steps to contribute` to this project. Also, add a CONTRIBUTING.md file to list down all the requirements required and the guidelines to follow while contributing to this project.

### PR DESCRIPTION
**Pull request by thejaysharma**
Made some changes to the readme.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (akshatnema) [inline CONTRIBUTING.md]:
```suggestion
```

**Review by akshatnema — CHANGES_REQUESTED** [review]
It's a good contribution @thejaysharma. Well done mate. There are just some small changes and then we are ready to merge it. Here are these:
- Remove the `CONTRIBUTING.md` file from this PR as it is being created and updated in the other one.
- Change the directory of `assets` folder from the server to web and move the folder inside the web directory.

File: CONTRIBUTING.md (hunk 1 above)
Comment (thejaysharma) [reply CONTRIBUTING.md]: Okay Sir, I'll do it.

**Comment by thejaysharma** [comment]
Hello 
I have made the changes as per your review request, do let me know if any more changes need to be made ?? 😃

File: README.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+![CSS3](https://img.shields.io/badge/-CSS3-orange?color=264DE4&style=for-the-badge&logo=CSS3&logoColor=white&logoWidth=20)
+![Javascript](https://img.shields.io/badge/-javascript-white?style=for-the-badge&logo=javascript&logoColor=white&logoWidth=20&color=F1DB4E)
+![ReactJS](https://img.shields.io/badge/-React-orange?color=09D9FE&style=for-the-badge&logo=React&logoColor=white&logoWidth=20)
+![NPM](https://img.shields.io/badge/-NPM-brightgreen?color=DC2C34&style=for-the-badge&logo=NPM&logoColor=white&logoWidth=20)
+![NodeJS](https://img.shields.io/badge/-Node-orange?color=8BBF3F&style=for-the-badge&logo=NODE&logoColor=white&logoWidth=20)
+![API](https://img.shields.io/badge/-API-orange?color=ff69b4&style=for-the-badge&logo=API&logoColor=white&logoWidth=20)
+![MongoDB](https://img.shields.io/badge/-MongoDB-orange?color=ncf&style=for-the-badge&logo=MongoDB&logoColor=white&logoWidth=20)
+
+## Want to Contribute ? 🚀
+Check Out  <a href="CONTRIBUTING.md">CONTRIBUTING.md</a>
```
Comment (akshatnema) [inline README.md]:
```suggestion
Check Out  [CONTRIBUTING.md](/CONTRIBUTING.md)
```

**Review by akshatnema — APPROVED** [review]
LGTM @thejaysharma 🥳

**Pull request merged**